### DISCUSSION TITLE
Soldering pin headers

### DISCUSSION THREAD

**Discussion by squaretones** [shanteacontrols/OpenDeck#235] [Q&A]
Hi, 

I just got the OpenDeck M board and am stoked to make my first MIDI controller. I gather I need to solder pin headers onto the board because they no longer come with them already on the board. Can anyone give some advice to a noob on doing that? Any recommendations for which pin headers and which solder to use?

I just plan to connect eight potentiometers and eight buttons to the board, and possibly some LEDs later on.

Thanks!

**Comment by paradajz** [comment]
Keywords you are looking for are male/female 2.54" pin headers. You can find them in any electronic store (Digikey, Mouser, Farnell) or eBay, AliExpress, etc.

**Reply by squaretones** [reply]
Thanks!

**Comment by gh0stless** [comment]
It's best to get a Dupont crimping pliers also, with a connector set. You will need it.

**Reply by squaretones** [reply]
Thanks!

**Reply by eurorackworld** [reply]
or even easier, buy dupont cables with female connectors :P
